Monika After Story/game/script-farewells.rpy

```diff
@@ -1460,27 +1460,110 @@ init 5 python:
     )
 
 label bye_prompt_housework:
-    if mas_isMoniNormal(higher=True):
-        m 1eub "Doing your chores, [player]?"
-        m 1ekc "I would like to help you out, but there's not really much I can do since I'm stuck in here..."
-        m 3eka "Just make sure to come back as soon as you're done, okay?"
-        m 3hub "I'll be waiting here for you~"
-    elif mas_isMoniUpset():
-        m 2esc "Fine."
-        m 2tsc "At least you're doing something responsible."
-        m 2tfc "{cps=*2}...For once.{/cps}{nw}"
+#TODOS: Chance for upset/distressed to not want to go with you, if not I'd ideally like it to default to regular goodbye
+# How to set it to non-generic "we're home" greeting?
```

Comment:
This was actually a minor error on my part, I forgot to generalize this portion. Can be done pretty easily though so I'll do it in this PR.